cloudbuild.yaml

```diff
@@ -13,7 +13,7 @@ steps:
     id: Maven Package
     env:
       - TZ=Europe/London
-    args: ['-Dmaven.repo.local=/workspace/.m2/repository', 'package']
+    args: ['-Dmaven.repo.local=/workspace/.m2/repository', 'verify', 'package']
```

Comment:
package no longer needed surely?
verify phase will be obliged to run all the preceeding phases which includes package

Reply:
good spot - redundant target removed